openmetadata-spec/src/main/resources/json/schema/api/data/createDataModel.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$id": "https://open-metadata.org/schema/api/data/createDataModel.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "CreateDataModelRequest",
+  "description": "Create Data Model entity request.",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.api.data.CreateDataModel",
+  "javaInterfaces": ["org.openmetadata.schema.CreateEntity"],
+  "properties": {
+    "name": {
+      "description": "Name that identifies this data model.",
+      "$ref": "../../type/basic.json#/definitions/entityName"
+    },
+    "displayName": {
+      "description": "Display Name that identifies this data model. It could be title or label from the source services.",
+      "type": "string"
+    },
+    "description": {
+      "description": "Description of the data model instance. What it has and how to use it.",
+      "$ref": "../../type/basic.json#/definitions/markdown"
+    },
+    "tags": {
+      "description": "Tags for this data model.",
+      "type": "array",
+      "items": {
+        "$ref": "../../type/tagLabel.json"
+      },
+      "default": null
+    },
+    "owner": {
+      "description": "Owner of this data model.",
+      "$ref": "../../type/entityReference.json"
+    },
+    "service": {
+      "description": "Link to the data model service where this data model is hosted in.",
+      "$ref": "../../type/basic.json#/definitions/fullyQualifiedEntityName"
+    },
+    "serviceType": {
+      "description": "Service type where this data model is hosted in.",
+      "$ref": "../../entity/services/dashboardService.json#/definitions/dashboardServiceType"
```

Comment:
Are the data models coming from regular databases or do dashboard services create there own datasets to power the dashboard?

Reply:
they are coming from databases. Its more like a collection, when they create a view it is part of DB https://powerbi.microsoft.com/en-us/what-is-data-modeling/